### PR TITLE
Test for CASSANDRA-10382

### PR DESCRIPTION
**Pull request by stef1927**

### CODE REVIEW

File: nodetool_test.py [hunk 1]
```diff
@@ -29,3 +31,34 @@ def test_decommission_after_drain_is_invalid(self):
...
+        """
+
+        cluster = self.cluster
+        cluster.populate([2, 2])
+        cluster.set_configuration_options(values={'endpoint_snitch': 'org.apache.cassandra.locator.GossipingPropertyFileSnitch'})
+
+        for i, node in enumerate(cluster.nodelist()):
+            with open(os.path.join(node.get_conf_dir(), 'cassandra-rackdc.properties'), 'w') as snitch_file:
+                for line in ["dc={}".format(node.data_center), "rack=rack{}".format(i % 2)]:
+                    snitch_file.write(line + "\n")
```
Comment (ptnapoleon) [inline nodetool_test.py]: Should we use `os.linesep` here?

**Comment by stef1927** [comment]
Fixed the linefeed separator thank you.

File: nodetool_test.py [hunk 2]
```diff
@@ -29,3 +31,34 @@ def test_decommission_after_drain_is_invalid(self):
...
+            self.assertEqual(0, len(err), err)
+            debug(out)
+            for line in out.split(os.linesep):
+                if line.startswith('Data Center'):
+                    self.assertTrue(line.endswith(node.data_center),
+                                    "Expected dc {} for {} but got {}".format(node.data_center, node.address(), line.rsplit(None, 1)[-1]))
+                elif line.startswith('Rack'):
+                    rack = "rack{}".format(i % 2)
+                    self.assertTrue(line.endswith(rack),
+                                    "Expected rack {} for {} but got {}".format(rack, node.address(), line.rsplit(None, 1)[-1]))
```
Comment (ptnapoleon) [inline nodetool_test.py]: Why not just return the entire line here and line 60?

Comment (stef1927) [inline nodetool_test.py]: That was the first attempt but it looked weird:  _Expected dc name for 127.0.0.x but got Data Center:                dcname_ so I extracted the dc name instead.

Comment (ptnapoleon) [inline nodetool_test.py]: Makes sense.

**Comment by ptnapoleon** [comment]
+1

**Comment by stef1927** [comment]
Thanks, ticket was committed so merging.

**Pull request merged**